newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -68,6 +80,20 @@ export class ObjectEditorDialog extends Component<*, StateType> {
         }
         titleStyle={styles.titleContainer}
       >
+        <div style={styles.container}>
```

Comment:
Instead of making a <div> with styling, use the components from `Grid.js`. In particular, check the `<Line>` component to replace this div.
The advantage is that this Line component provides the margin, so you're sure that you have the same consistent margins as in the rest of the software :)

Reply:
these built in layout tools are very cool. I was looking for some documentation on them, but the editor itself is already full of great examples